x/tx/signing/textual/types.go

```diff
@@ -46,9 +48,12 @@ type ValueRenderer interface {
 type RepeatedValueRenderer interface {
 	ValueRenderer
 
-	// FormatRepeated should render the value to a text plus annotation.
+	// Format renders the Protobuf list value to a list of Screens.
 	FormatRepeated(context.Context, protoreflect.Value) ([]Screen, error)
 
-	// ParseRepeated should be the inverse of Format.  The list will be populated with the repeated values.
+	// Parse is the inverse of Format. It must be able to parse all valid
+	// screens, however the behavior of Parse on invalid screens is not
+	// specified, and does not necessarily error. The `protoreflect.List`
+	// argument will be mutated and populated with the repeated values.
 	ParseRepeated(context.Context, []Screen, protoreflect.List) error
```

Comment:
```suggestion
	// ParseRepeated is the inverse of FormatRepeated. It must parse all
	// valid screens, however the behavior on invalid screens is not
	// specified, and does not necessarily error. The `protoreflect.List`
	// argument will be mutated and populated with the repeated values.
	ParseRepeated(context.Context, []Screen, protoreflect.List) error
```

Reply:
If a method (ParseRepeated) returns a nil error, then callers will assume the execution was successful. If "the behavior on invalid screens is not specified" then this is a problem, invalid screens must result in a non-nil error return.

Reply:
Haven't gone through all renderers in detail yet but as far as I can see we are always returning an error if something goes wrong. I'll bring this up in the work group call, I remember that there was a reason for `the behavior on invalid screens is not specified` but don't remember exactly why.

Reply:
Parse is used for testing, so we don't plan to spend much effort on these functions